index.html

```diff
@@ -1,10 +1,13 @@
 <!doctype html>
 
+<meta name="viewport" content="width=device-width, initial-scale=1.0">
 <html lang="en">
 <head>
   <meta charset="utf-8">
 
-  <title>SpaceWalkers</title>
+  <title> 
+    <h1>SpaceWalkers</h1>
+</title>
 
```

Comment:
We went over this in our 1:1, but remember to put the `<meta`> tag inside the `<head>` tags, and `<title>` is for the browser's tab title